Paintroid/src/main/res/layout/dialog_pocketpaint_text_tool.xml

```diff
@@ -125,18 +126,38 @@
 
         <View style="@style/PocketPaintDividerHorizontal"/>
 
-        <EditText
-            android:id="@+id/pocketpaint_text_tool_dialog_input_text"
+
+        <LinearLayout
+            android:orientation="horizontal"
             android:layout_width="match_parent"
+            android:weightSum="4.0"
```

Comment:
If you remove the weightsum, then android will calculate this sum itself. (So if you have 2 child views, with `layout_weight="2"`, the sum will be 4). Normally it's better to just remove the `weightSum` and let android do it.